package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@tabit/rabbitmq",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

```suggestion
  "version": "3.1.0",
```

We mostly use patch versions to indicate hotfixes, or fixes in general.